### DISCUSSION TITLE
NetBox Labs is hiring a NetBox software engineer

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#13959] [Announcements]
NetBox Labs is looking for a Software Engineer with 5+ years of experience to help us accelerate delivery of features and functionality into the NetBox application for both open source NetBox and NetBox Cloud. This role will work closely with both the core NetBox maintainers and the NetBox Labs platform team to execute on high value NetBox features.

Check out the full [US](https://netboxlabs.com/careers/software-engineer-netbox-us/) or [UK/EU](https://netboxlabs.com/careers/software-engineer-netbox-ukeu) job description, and get in touch with us at careers@netboxlabs.com if you think you'd be good fit!

**Comment by jsenecal** [comment]
Hey @jeremystretch , FYI the links are 404 😮

**Comment by jcollie** [comment]
Perhaps because they have hired someone?

Jeff

On Mon, Oct 23, 2023 at 5:13 PM Jonathan Senecal ***@***.***>
wrote:

> Hey @jeremystretch <https://github.com/jeremystretch> , FYI the links are
> 404 😮
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/netbox-community/netbox/discussions/13959#discussioncomment-7363758>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFUVNVBIGHQNDCBWBX4DYDYA3TZ3AVCNFSM6AAAAAA5RMULHCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNRTG42TQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <netbox-community/netbox/repo-discussions/13959/comments/7363758@
> github.com>
>


-- 
Jeff Ollie
The majestik møøse is one of the mäni interesting furry animals in Sweden.

**Comment by mtinberg** [comment]
I'm assuming the link is now here
https://netboxlabs.com/job/software-engineer-netbox-us-remote/
or https://netboxlabs.com/job/senior-software-engineer-us/

and not https://netboxlabs.com/careers/software-engineer-netbox-us/

I just edited the URL to remove the last part and was taken to the /careers/ page which listed the openings.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
